### PR TITLE
v2.3.2, no error for single batch

### PR DESCRIPTION
**Pull request by Alanocallaghan**
Same as the other PR but for devel

### CODE REVIEW

**Comment by catavallejos** [comment]
@Alanocallaghan just realised there was a second PR. Is this ready to be merged too? The two check errors are due to a small difference in the coverage of the unit tests, so I think it should be ok. 

Before I merge, could you please update `NEWS`?

**Comment by Alanocallaghan** [comment]
Yep, same content. The other was for release, this is devel (just bug fixes in both cases). Done

File: DESCRIPTION [hunk 1]
```diff
@@ -1,12 +1,12 @@
 Package: BASiCS
 Type: Package
 Title: Bayesian Analysis of Single-Cell Sequencing data
-Version: 2.3.1
-Date: 2020-12-14
-Authors@R: c(person("Catalina", "Vallejos", role=c("aut", "cre"),
+Version: 2.3.2
+Date: 2020-02-25
```
Comment (catavallejos) [inline DESCRIPTION:5]: @Alanocallaghan can you update this to be 2021-04-14 ?

File: NEWS [hunk 2]
```diff
@@ -1564,3 +1564,6 @@ version 2.3.1 (2020-12-14)
 when using an EB prior for mu
 - lower minimum tolerance `mintol_mu` (1e-5 instead of 1e-3) as a default value
 in `.BASiCS_MCMC_ExtraArgs`
+
+version 2.3.2 (2020-04-14)
```
Comment (catavallejos) [inline NEWS:1568]: Same here. Date to be updated.

Comment (Alanocallaghan) [reply NEWS:1568]: lol definitely living in 2020 still

Comment (catavallejos) [reply NEWS]: time has a weird dimension these days ...

**Pull request merged**